recipes/open62541/all/patches/set_static_version_v_1_0_1.patch

```diff
@@ -0,0 +1,589 @@
++    0,     /* .maxMessageSize, 0 -> unlimited */
++    0      /* .maxChunkCount, 0 -> unlimited */
+ };
+ 
+ /***************************/
+@@ -57,10 +57,6 @@ const UA_ConnectionConfig UA_ConnectionConfig_default = {
+ #define APPLICATION_URI "urn:unconfigured:application"
+ #define APPLICATION_URI_SERVER "urn:open62541.server.application"
+ 
+-#define STRINGIFY(arg) #arg
+-#define VERSION(MAJOR, MINOR, PATCH, LABEL) \
+-    STRINGIFY(MAJOR) "." STRINGIFY(MINOR) "." STRINGIFY(PATCH) LABEL
+-
+ static UA_StatusCode
+ createEndpoint(UA_ServerConfig *conf, UA_EndpointDescription *endpoint,
+                const UA_SecurityPolicy *securityPolicy,
+@@ -69,17 +65,16 @@ createEndpoint(UA_ServerConfig *conf, UA_EndpointDescription *endpoint,
+ 
+     endpoint->securityMode = securityMode;
+     UA_String_copy(&securityPolicy->policyUri, &endpoint->securityPolicyUri);
+-    endpoint->transportProfileUri =
+-        UA_STRING_ALLOC("http://opcfoundation.org/UA-Profile/Transport/uatcp-uasc-uabinary");
++    endpoint->transportProfileUri = UA_STRING_ALLOC(
++        "http://opcfoundation.org/UA-Profile/Transport/uatcp-uasc-uabinary");
+ 
+     /* Add security level value for the corresponding message security mode */
+-    endpoint->securityLevel = (UA_Byte) securityMode;
++    endpoint->securityLevel = (UA_Byte)securityMode;
+ 
+     /* Enable all login mechanisms from the access control plugin  */
+-    UA_StatusCode retval = UA_Array_copy(conf->accessControl.userTokenPolicies,
+-                                         conf->accessControl.userTokenPoliciesSize,
+-                                         (void **)&endpoint->userIdentityTokens,
+-                                         &UA_TYPES[UA_TYPES_USERTOKENPOLICY]);
++    UA_StatusCode retval = UA_Array_copy(
++        conf->accessControl.userTokenPolicies, conf->accessControl.userTokenPoliciesSize,
++        (void **)&endpoint->userIdentityTokens, &UA_TYPES[UA_TYPES_USERTOKENPOLICY]);
+     if(retval != UA_STATUSCODE_GOOD)
+         return retval;
+     endpoint->userIdentityTokensSize = conf->accessControl.userTokenPoliciesSize;
+@@ -97,7 +92,7 @@ static UA_UsernamePasswordLogin usernamePasswords[2] = {
+ 
+ static UA_StatusCode
+ setDefaultConfig(UA_ServerConfig *conf) {
+-    if (!conf)
++    if(!conf)
+         return UA_STATUSCODE_BADINVALIDARGUMENT;
+ 
+     /* Zero out.. All members have a valid initial value */
+@@ -114,14 +109,12 @@ setDefaultConfig(UA_ServerConfig *conf) {
+     conf->buildInfo.productUri = UA_STRING_ALLOC(PRODUCT_URI);
+     conf->buildInfo.manufacturerName = UA_STRING_ALLOC(MANUFACTURER_NAME);
+     conf->buildInfo.productName = UA_STRING_ALLOC(PRODUCT_NAME);
+-    conf->buildInfo.softwareVersion =
+-        UA_STRING_ALLOC(VERSION(UA_OPEN62541_VER_MAJOR, UA_OPEN62541_VER_MINOR,
+-                                UA_OPEN62541_VER_PATCH, UA_OPEN62541_VER_LABEL));
+-    #ifdef UA_PACK_DEBIAN
++    conf->buildInfo.softwareVersion = UA_STRING_ALLOC("1.0.1");
```

Comment:
I see only few changes on these patches, I think they could be reduced.

Reply:
I merged both versions of `set_static_version_*.patch` into a single one, however I dont see how I could do the same for `add_conan_center_index_compliance_*.patch` since root **CMakeLists.txt** was slightly  changed in between the two versions. I expermented with one mega patch for both versions, but it always ended up with `git: patch does not apply` error.